metrics/ci_integrations/docs/source/github_actions/config/configuration_template.yaml

```
@@ -0,0 +1,33 @@
+# A template for ci_integrations config with the Github Actions integration as a `source`.
+# Consider the following links to familiarize yourself with Github Actions:
+# - https://docs.github.com/en/free-pro-team@latest/rest/reference/actions
+# - https://docs.github.com/en/free-pro-team@latest/actions/reference/authentication-in-a-workflow
+
+source:
+  # The `github_actions` key defines that the following source config stands for Github Actions integration.
+  github_actions:
+
+    # A unique identifier of the Github Actions workflow to integrate.
+    #
+    # This identifier is used to determine a workflow from where we will get new builds.
+    #
+    # You can get it from your Github link. For example: `https://github.com/{repository_owner}/{repository_name}/actions/runs/{workflow_id}`
```

```suggestion
    # A unique identifier of the Github Actions workflow.
    # 
    # The tool uses this identifier to fetch builds. 
    # This can be either the workflow ID or a name of the file that defines this workflow 
    # (for example, `web_actions.yaml`). You can surely use the name of your file as it is 
    # handier and clearer. To obtain the ID of the desired workflow, consider using the 
    # GitHub Actions REST API and list your repository workflows: 
    # - https://docs.github.com/en/free-pro-team@latest/rest/reference/actions#list-repository-workflows.
```
The link `https://github.com/{repository_owner}/{repository_name}/actions/runs/{workflow_id}` is actually `https://github.com/{repository_owner}/{repository_name}/actions/runs/{run_id}`.